devices/Paexo/conf/paexo-validation-motor-control.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
 # set n to 1
-min_motor_pos=27
-max_motor_pos=55
+min_motor_pos=30
+max_motor_pos=50
 step_size=2
 
 ## Paexo initialization
 echo "start" | yarp rpc /wearable/paexo/rpc:i
 sleep 0.5
 echo "en_bc_data" | yarp rpc /wearable/paexo/rpc:i
 sleep 0.5
-echo "init:r" | yarp rpc /wearable/paexo/rpc:i
+echo "init:l" | yarp rpc /wearable/paexo/rpc:i
```

Comment:
If I am not wrong, we should now have moved to buffered port and not rpc? Or is the rpc command still working?

Reply:
The rpc feature is still there, as it is also used for other commands for Paexo. This file is the shell script we used during FT identification. I will leave to right, as we switched that actuator to right.

Reply:
Ok, great, I was not sure if the `rpc` port was remaining or not.

Reply:
Done in https://github.com/robotology/wearables/pull/124/commits/faefcae8f616d0b1a912f78e50a7340c42e037a0